cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/InterpretedT3xModel.java

```diff
@@ -66,6 +66,7 @@ static class InterpretedWinding {
         final TapChangerConversion.InterpretedEnd end1;
         final InterpretedEnd2 end2;
         final boolean structuralRatioAtEnd2;
+        final double ratedS;
```

Comment:
@zamarrenolm we have a regression here because of this `double` instead of `Double`. I am going to open a PR for this bug fix and it will lead to a patch release of core.